plugins/discourse-ai/lib/translation/category_candidates.rb

```diff
@@ -11,13 +11,32 @@ def self.get
         categories
       end
 
-      def self.get_completion_per_locale(locale)
-        done = get.where(locale:).count
-        done += CategoryLocalization.where(locale:).count
+      private
 
-        total = get.count
+      def self.calculate_completion_per_locale(locale)
```

Comment:
Still a bit of duplication here per-type but we can de-duplicate when we start finalising other models such as tags or localizing tables from other plugins such as post voting comments.